### DISCUSSION TITLE
how to backup/ restore chroma data for vanna.ai

### DISCUSSION THREAD

**Discussion by njalan** [vanna-ai/vanna#402] [Q&A] [answered]
If I want to backup and restore the training data to another vanna instance, is there any guidence?
Where is the data location for  chroma data?

**Comment by andreped** [comment, marked as answer]
Hello @njalan! :] This sounds more like a discussion topic than an issue.

Previously, I have created the `chroma.sqlite3` file by training on relevant data and stored it in an Azure storage. Then for deployment of a container, I download the `chroma.sqlite3` file into the container and point to it for the persistent storage when initializing the Chroma/Vanna instance. That way I always have backup and I avoid training during app launch.

**Comment by njalan** [comment]
@andreped  Thanks for your reply and it is working when use chroma.sqlite3

**Reply by andreped** [reply]
@njalan Would be great if you could mark my suggestion as **the answer**. Cheers! :]

**Comment by guyzilb** [comment]
can you share how you use chromadb as vectordb please ?

**Reply by zainhoda** [reply]
https://vanna.ai/docs/postgres-openai-standard-chromadb/

**Reply by guyzilb** [reply]
@zainhoda 

but which config I should pass? 
I have choromadb in my local 

class MyVanna(ChromaDB_VectorStore, OpenAI_Chat):
    def __init__(self, config=None):
        ChromaDB_VectorStore.__init__(self, config=config)
        OpenAI_Chat.__init__(self, config=config)

vn = MyVanna(config={'api_key': '************************', 'model': 'gpt-4'})

what I missed?

**Comment by rinshadka** [comment]
how to point the db file to an alternate path?
